contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -284,6 +284,14 @@ contract OETHVaultCore is VaultCore {
     }
 
     function _claimWithdrawal(uint256 requestId)
+        internal
+        virtual
+        returns (uint256 amount)
+    {
+        return _claimWithdrawal(requestId, CLAIM_DELAY);
+    }
+
+    function _claimWithdrawal(uint256 requestId, uint256 claimDelay)
```

Comment:
I assume a second internal function was added so Mainnet can be backward compatible with using the `CLAIM_DELAY` constant? I'd be inclined to also change mainnet to use the new `withdrawalClaimDelay` storage variable the next time the vault is upgraded so we don't have code to handle both `CLAIM_DELAY` and `withdrawalClaimDelay`.

Reply:
> I assume a second internal function was added so Mainnet can be backward compatible with using the CLAIM_DELAY constant?

Yes, that was the reason.

> I'd be inclined to also change mainnet to use the new withdrawalClaimDelay storage variable the next time the vault is upgraded so we don't have code to handle both CLAIM_DELAY and withdrawalClaimDelay.

I assumed that we would not make it gas expensive. But I guess it should be negligible and will keep our codebase clean, will make this change, thanks

Reply:
Have made that change